tox.ini

```
@@ -162,9 +162,10 @@ envlist =
     kafka-messagebroker_confluentkafka-{py27,py39}-confluentkafka{0107,0106},
     ; confluent-kafka had a bug in 1.8.2's setup.py file which was incompatible with 2.7.
     kafka-messagebroker_confluentkafka-{py39}-confluentkafka{0108},
-    kafka-messagebroker_kafkapython-{pypy27,py27,py37,py38,pypy37}-kafkapythonlatest,
+    kafka-messagebroker_kafkapython-{pypy,py27,py37,py38,pypy37}-kafkapythonlatest,
```

```suggestion
    kafka-messagebroker_kafkapython-{pypy27,py27,py37,py38,pypy37}-kafkapythonlatest,
```
Put this line back, or pypy27 will have difficulties on tox v4. And particularly my M1 Mac.